index.bs

```diff
@@ -12188,18 +12190,19 @@ internal method as follows.
 
 <h4 id="legacy-platform-object-abstract-ops">Abstract operations</h4>
 
-<div algorithm>
+<div algorithm="to determine if a property name is an array index">
```

Comment:
You don't need a value for the "algorithm" attribute if there's a DFN in it. (Ideally, all algorithm would have a dfn and wouldn't require anything but a boolean "algorithm" attribute.

Reply:
If I leave it out, the generated `data-algorithm` attribute has `[=array index=]` instead of `array index`. We can regard that a yet to be fixed Bikeshed bug though. @tabatkins

Reply:
Oh, that might be because it can't disambiguate the two names in the "lt" attribute. Just leave in the "algorithm" attribute, then.

Reply:
Nah, that looks like a bug on my part. Curious!